.drone.yml

```diff
@@ -0,0 +1,111 @@
+---
+kind: pipeline
+name: preview
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: install
+    image: node:18-alpine
+    commands:
+      - yarn install --frozen-lockfile
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
```

Comment:
do we need to mount the docker volumes for those steps? and same for the below ones.